applications/convection_diffusion_application/python_scripts/conjugate_heat_transfer_solver.py

```diff
@@ -0,0 +1,305 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+import sys
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("FluidDynamicsApplication")
+KratosMultiphysics.CheckRegisteredApplications("ConvectionDiffusionApplication")
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+import KratosMultiphysics.ConvectionDiffusionApplication as KratosConvDiff
+# TODO: USE THE MAPPING APPLICATION INSTEAD OF THE FSI MAPPER
+import KratosMultiphysics.FSIApplication as FSIApplication
+import NonConformant_OneSideMap as ncosm
```

Comment:
Why not using mortar mapper?

Reply:
We can discuss about that, however this is best done in a separate PR. Why do we have both Mortar Mapper and the NonConformant whatever map? do they at least have the same interface?

i would open an issue to discuss about that

Reply:
As you see I am not blocking, just asking. The interface is similar, but not the same. The mortar mappper converges always in one iteration (maximum two), because solves exactly the problem

Reply:
I believe there is even an application for this ...

Does not yet have the fancy mappers yet though

Reply:
as i am telling, you have a point. open an issue and we can discuss that in general...not as part of this PR in any case

Reply:
Ok